docs/sources/explore/_index.md

```diff
@@ -10,159 +10,53 @@ labels:
     - cloud
     - enterprise
     - oss
+menuTitle: Explore
```

Comment:
Is there documentation about the explore apps somewhere? We get a lot of confusion between the original explore section and the apps, having this addressed with even a "coming soon" with some expectation might be good

Reply:
Yes, it's under Simplified Exploration in Cloud docs, and it's being moved (by me) into a separate folder in OSS. I have to create a design doc for the information architecture reorg.

Reply:
I'm sorry, I don't understand why your future work prohibits you from adding information into this landing page.

Reply:
I am not sure what you mean by that comment. The Explore apps have existing documentation and will have a separate landing page, that's being addressed as part of a larger information architecture reorg.  The Explore landing page pertains to Explore page-related docs. We have specific guidelines that I need to follow regarding what info goes into landing pages so that our docs look cohesive.  The current temporary landing page for Explore apps is: https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration.

Reply:
Thank you for the context. I think with the very common confusion that happens between the two sections, it's worth a discussion if an exception can be made with some information to aid people who come to this landing page expecting information about the explore apps.

Reply:
As I stated, this is being addressed as part of a larger project currently in progress due next month.

Reply:
As I stated, I do not understand why the future work means you cannot add information to this landing page in this PR.